lib/ansible/modules/cloud/azure/azure_rm_manageddisk_info.py

```diff
@@ -228,7 +229,7 @@ def list_items_by_resource_group(self):
 
         results = []
         for item in response:
-            if self.has_tags(item.tags, self.tags):
+            if self.has_tags(item.tags, eval(self.tags)):
```

Comment:
This change does not look good for me. Is it possible to change the type for tags at first?

Reply:
Do you think I need to import the json module and convert the type before using it?

Reply:
Do you think I need to import the json module and convert the type before using it?

Reply:
The tags parameter should be parsed as list like below ? https://github.com/ansible/ansible/blob/b48a89682bf9d84f12110ba1bff125a6978375fd/lib/ansible/modules/cloud/azure/azure_rm_dnszone_info.py#L148